spec/openapi.yaml

```diff
@@ -8358,7 +8427,212 @@ paths:
             -H "Authorization: Bearer $smartlingToken" \
             -H "Content-Type: application/json" \
             -d '{ "action": "execute" }'
+  /jobs-batch-api/v2/projects/{projectId}/batches/{batchUid}/file:
+    post:
+      summary: Add file to a batch asynchrounosly
+      description: >
+        This endpoint is actually a proxy for Upload File in the Files API, and
+        it works with these two differences:
+         1. It additionally gets `batchUid` as a path parameter.
+         2. It doesn't relay `localeIdsToAuthorize` to Files Api, therefore the file stays unauthorized. Instead, `localeIdsToAuthorize` are saved just for this batch session and are used when a file is being attached to a job.
+
+        When the first file is added to the new batch, it becomes an
+        `ADDING_FILES` status. Adds original source content to a Job batch in
```

Comment:
Change "it becomes an 'adding_files' status" to "the batch changes to 'ADDING_FILES' status"